src/database/migrations/20190703214918-create-article.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
I notice you already have three migrations for the `Articles` model, is there are reason why?